frontend/packages/dev-console/src/components/deployments/DeploymentConfigPage.tsx

```diff
@@ -13,7 +13,7 @@ const DeploymentConfigPage: React.FC = () => {
   const { t } = useTranslation();
   const { ns: namespace, name } = useParams();
 
-  const isNew = !name || name === '~new';
```

Comment:
I think this one is actually valid, since the `useParams` returns URL fragments, where the name of the DC could actually be `~new`

Reply:
`useParams` returns the following when creating a new `DeploymentConfig`:
```
{
    "*": "k8s/ns/default/deploymentconfigs/~new/form",
    "ns": "default"
}
```

Reply:
https://github.com/openshift/console/blob/585e061fbeb9ab4eb1d400cce435dc8a81f8290f/frontend/packages/dev-console/console-extensions.json#L1203 would explain that.  There is no parameter in the path for create.